thorlcr/msort/tsorts1.cpp

```diff
@@ -55,15 +55,37 @@ class CMergeReadStream : public CSimpleInterface, public IRowStream
     }
 public:
     IMPLEMENT_IINTERFACE_USING(CSimpleInterface);
-    CMergeReadStream(IThorRowInterfaces *rowif, unsigned streamno,SocketEndpoint &targetep, rowcount_t startrec, rowcount_t numrecs)
+    CMergeReadStream(IThorRowInterfaces *rowif, unsigned streamno,SocketEndpoint &targetep, rowcount_t startrec, rowcount_t numrecs, bool useTLS=false, unsigned sortTraceLevel=0, ISecureSocketContext *secureContextClient=nullptr)
```

Comment:
It looks like it would be enough to pass in a ISecureSocketContext , to signify TLS, i.e. don't need both useTLS and a ISecureSocketContext on ctor.